tools/end2end_tests/images/branch/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM dart:latest AS buildimage
```

Comment:
This should also be pinned e.g. `dart:3.0.5@sha256:65e5f5d6d72ad2f7b32f402c01b5fe8a426455b1ede1e9f840f95a2a8c14afbd`

Reply:
https://github.com/atsign-foundation/sshnoports/pull/212/commits/bd964acdcd1a038af73ac6c89df250a2086aa6ff